src/ser.rs

```diff
@@ -1611,18 +1384,18 @@ impl CharEscape {
 pub trait Formatter {
     /// Writes a `null` value to the specified writer.
     #[inline]
-    fn write_null<W: ?Sized>(&mut self, writer: &mut W) -> io::Result<()>
+    fn write_null<W: ?Sized>(&mut self, writer: &mut W) -> Result<()>
```

Comment:
This is a breaking change. Please avoid making this change.

The Formatter trait should not be included in the public API in no_std mode.